addons/halo-cluster/templates/cluster.yaml

```diff
@@ -10,4 +10,9 @@
         type: Noop
       {{- include "kblib.componentResources" . | indent 6 }}
       {{- include "kblib.componentStorages" . | indent 6 }}
-      {{- include "kblib.componentServices" . | indent 6 }}
\ No newline at end of file
+      {{- include "kblib.componentServices" . | indent 6 }}
+      serviceRefs:
+      - name: etcdService
+        namespace: default
+        cluster: kb-etcd-for-halo
```

Comment:
etcd cluster name is hard-coded? I'd susget use some var defined in `values.yaml' file.

Reply:
now I have changed the Cluster architecture of halo-cluster , independence the service etcd